.gitlab/binary_build/system_probe.yml

```diff
@@ -11,6 +11,7 @@
     - find "$CI_BUILDS_DIR" ! -path '*DataDog/datadog-agent*' -depth  # -delete implies -depth
     - find "$CI_BUILDS_DIR" ! -path '*DataDog/datadog-agent*' -delete || true  # Allow failure, we can't remove parent folders of datadog-agent
   script:
+    - source /root/.bashrc
```

Comment:
Is this necessary? I feel like sourcing bashrc should be part of the Docker image build, no? Asking to check if this is a bug in how we build and configure that system-probe image.

Reply:
Basically this is needed if the runner doesn't use a login shell, which don't source `.bashrc`

AFAIK this is needed for k8s runners at least

Reply:
I triggered a CI pipeline with KMT jobs enabled (the RUN_KMT_TESTS variable is not enabled by default) just to check if this happens to more system-probe jobs (sorry about the comment spam :P)

Reply:
It looks like there are failures in other system-probe jobs, mainly the KMT ones ([example](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/476872465)), with the `aws` command not being found. I think we should make sure that the conda env is activated for non-login shell too. I can make a PR for that and test it needed.

Reply:
Indeed @gjulianm if this PR fixes system probe tests, my revert PR is unnecessary